signac/_synced_collections/backends/collection_json.py

```diff
@@ -116,6 +117,16 @@ def json_attr_dict_validator(data):
 """
 
 
+class _IsWindows:
+    """A truthy type that is False on Windows and True otherwise."""
+
+    def __bool__(self):
+        if sys.platform.startswith("win32") or sys.platform.startswith("cygin"):
+            return False
+        else:
+            return True
```

Comment:
Assuming it should be `True` on windows and I think there is a typo:
```suggestion
        return sys.platform.startswith("win32") or sys.platform.startswith("cygwin"):
```

Reply:
See the discussion below.